dace/data.py

```diff
@@ -136,27 +136,6 @@ def create_datadescriptor(obj, no_custom_desc=False):
                     'adaptor method to the type hint or object itself.')
 
 
-def find_new_name(name: str, existing_names: Sequence[str]) -> str:
```

Comment:
I think that this is part of the external API, so we cannot remove it. It could be moved to a clearer utility file, but maybe this has to be discussed in a DaCe meeting.

Reply:
Nvm, it is duplicate.